.travis.yml

```diff
@@ -21,7 +29,7 @@ before_script:
 
 after_success:
   - >
-      test "${TRAVIS_PULL_REQUEST}" = 'false' && test "${TRAVIS_JDK_VERSION}" = 'openjdk6' &&
+      test "${TRAVIS_PULL_REQUEST}" = 'false' && test "${TRAVIS_JDK_VERSION}" = 'openjdk6' -o "${TRAVIS_SCALA_VERSION}" = '2.12.0-M4' &&
```

Comment:
The intent here is to enable publishing with Java 8 only for the Scala 2.12 build.